azure-pipelines.yml

```diff
@@ -21,7 +21,7 @@ stages:
               rustup_toolchain: stable
             linux-pinned:
               imageName: 'ubuntu-16.04'
-              rustup_toolchain: 1.43.0
+              rustup_toolchain: 1.45.0
```

Comment:
why do we need the bump btw?

Reply:
@Keats, please look at this https://github.com/getzola/zola/pull/1129#discussion_r480172320

Reply:
Wouldn't `trim_start_matches` do the same?

Reply:
In that case, I think yes. To be honest I used strip_prefix because I looked at the documentation and it was what I needed and I did not look further at it. 
As I used a version > 1.45, I have not realize that this method was not stable before that release.
Anyway, would you like me to use trim_start_matches instead of strip_prefix and come back to 1.43 for the pinned version ?

Reply:
If you have time and don't mind sure